### DISCUSSION TITLE
I want my special user repository to show images how can I do this?

### DISCUSSION THREAD

**Discussion by On-Fi** [community/community#84693] [Repositories] [answered]
### Select Topic Area

Question

### Body

I want my special user repository to show memes like this cool developer did:
https://github.com/dingdingwall
(like under his headline "**😂 Random Dev Meme**")
<br>
I made a repo (**using html, css and js**) that renders a meme every time you reload the page:
https://github.com/On-Fi/codermemes
<br>
I uploaded this repo on [vercel](https://codermemes.vercel.app/) and linked it into my special user repository, <br> like this: <br>`<img src='https://codermemes.vercel.app/' style="height: 400px;"/>`
but it doesn't work, like it works on @dingdingwall's repo.  
<br>
I looked at @dingdingwall readme and clicked his vercel link and saw that the url says "camo.githubusercontent.com",
I when googled what camo.github is and found this link:
https://github.com/atmos/camo
<br>
*I'm new to coding and don't understand how to use camo.*
**Can somebody please help me?
Do I have to rewrite my app in a different language? And if that is the case how do I do that?**

**Comment by darkotodoric** [comment]
His website (https://randommeme-five.vercel.app/) opens a random image. Your website (https://codermemes.vercel.app/) opens HTML with a random image inside HTML. Change that and it will work.

**Reply by On-Fi** [reply]
Okay cool thank you @darkotodoric for your answer.
And how do I change that? 
I only know how to use HTML, CSS and JS right now.

**Reply by darkotodoric** [reply, marked as answer]
Not sure how this "Vercel" works but as I see you need little more knowledge (Node.js). You can check [this](https://github.com/PepeBigotes/random-image) repository, probably [dingdingwall](https://github.com/dingdingwall) guy doing similar stuff.

**Reply by On-Fi** [reply]
Thank You for your help, Darko :)
and "Vercel" is used for hosting like "Github Pages".

**Comment by dingdingwall** [comment]
bro, I just used a github profile generator. I think the owner used api for the memes, just feel free to use my repo it's not mine anyway but thank you for calling me "cool developer" lol I'm also a new developer like you.  Have I answered your question? @On-Fi

**Reply by On-Fi** [reply]
Thank You Bro that you explained your process.
the fact that you answered my question made you even cooler in my eyes ;) hehe
